packages/woly/package.json

```
@@ -44,7 +44,7 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/woly-ui/woly",
-    "directory": "packages/calendar"
+    "directory": "packages/upload"
```

```suggestion
    "directory": "packages/woly"
```

Это ж файл описания woly пакета